controllers/recipes/recipes-schema.js

```diff
@@ -0,0 +1,19 @@
+import mongoose, {Schema} from 'mongoose';
+
+const RecipesSchema = mongoose.Schema({
+    authorId: {type: Schema.Types.ObjectId, ref: "UserModel",required: true},
```

Comment:
I think we might need to change this to "UsersModel" since our users-model.js calls it that

Reply:
Also tested it on postman and everything works. The same suggestion as Macee's about changing UserModel to UsersModel